content/challenges/easy/7 - Singular or Plural/func.js

```diff
@@ -9,5 +9,16 @@
  * @returns {boolean}
  */
 module.exports = (word) => {
-    
+    // if (word.endsWith("s")) {
+    //     return true;
+    // } else {
+    //     return false;
+    // } 
+    // passed all but need to break it down more
+    // if (word[word.length - 1] === "s") {
+    //     return true;
+    // } else {
+    //     return false;
+    // }
+    return (word[word.length - 1] === 's');
```

Comment:
this is one of the rare cases where this is the right move.

A single conditional check is fine, in my opinion, in a return statement. Compound conditionals not so much!